### PR TITLE
feat(cloudbuilder): Add UsePersistedStoragePath configuration 

### LINKED ISSUE #636: Expose `UsePath` in the CamundaCloudClientBuilder for persisted token storage.
**Is your feature request related to a problem? Please describe.**
Hello, recently started setting up lower test environments that point to a cloud instance of camunda 8 and ran into an issue where the library is attempting to persist tokens to disk for caching (`.zeebe`); however, this directory is read only in our environment and due to restrictions beyond my control I am unable to make it not read only; however, we do have a location that we would be able to write to.

I see there is a `UsePath` method on the `ICamundaCloudTokenProviderBuilderFinalStep` that isn't exposed via the `ICamundaCloudClientBuilder` and I don't see any other way of configuring the `CamundaCloudTokenProviderBuilder`.

**Describe the solution you'd like**
I would like to expose the `UsePath` on the the `ICamundaCloudClientBuilder` so we can properly configure where we'd like to persist our token cache.

**Describe alternatives you've considered**
Only alternative as of right now is to downgrade to before this change was implemented.

**Additional context**
No additional context.

### PR DESCRIPTION
**Pull request by ShawnAbshire**
Add UsePersistedStoragePath configuration to the final step of the cloud builder.

closes #636

### CODE REVIEW

File: Client/Api/Builder/ICamundaCloudClientBuilder.cs [hunk 1]
```diff
@@ -76,6 +76,13 @@ public interface ICamundaCloudClientBuilderFinalStep
         /// <returns>the next step in building a ICamundaCloudClient</returns>
         ICamundaCloudClientBuilderFinalStep UseAuthServer(string url);
 
+        /// <summary>
+        /// Sets the given path to store credentials on disk.
+        /// </summary>
+        /// <param name="path">the path where to store the credentials.</param>
+        /// <returns>the fluent ICamundaCloudClientBuilderFinalStep.</returns>
+        ICamundaCloudClientBuilderFinalStep UsePersistedStoragePath(string path);
```
Comment (ShawnAbshire) [inline Client/Api/Builder/ICamundaCloudClientBuilder.cs:84]: Thought it we be more ideal to be more explicit with what path you're settings.

Comment (Zelldon) [reply Client/Api/Builder/ICamundaCloudClientBuilder.cs:84]: Not sure what you mean here?

**Review by Zelldon — CHANGES_REQUESTED** [review]
@ShawnAbshire you need to sign our CLA otherwise I can't merge your PR.

File: Client/Api/Builder/ICamundaCloudClientBuilder.cs (hunk 1 above)
Comment (ShawnAbshire) [reply Client/Api/Builder/ICamundaCloudClientBuilder.cs:84]: Just meant thought it would be better than passing forward the `UsePath` name.

**Comment by ShawnAbshire** [comment]
I've already signed it before; however, I can't seem to sign it again and not sure how to resolve?  Image attached on what i'm seeing.  Going to create a new branch and resubmit the PR.


cla

**Pull request closed without merging**